src/main/java/htsjdk/variant/variantcontext/GenotypeNumLikelihoodsCache.java

```diff
@@ -0,0 +1,43 @@
+package htsjdk.variant.variantcontext;
+
+import java.util.HashMap;
+
+/**
+ * Hybrid dynamic cache for genotype likelihood counts
+ */
+
+public class GenotypeNumLikelihoodsCache {
+
+    private final static int N_ALLELES = 5;
```

Comment:
These should be specified in 1 place, either in GenotypeLikelihoodUtils or here but not both.

Reply:
I think I'll move them out of GenotypeLikelihoods so they're just here